tests/testStreamStop.cpp

```diff
@@ -91,6 +94,7 @@ class TestStreamStop : public ::testing::Test,
     AudioStreamBuilder mBuilder;
     AudioStream *mStream = nullptr;
     static constexpr int kTimeoutInNanos = 1000 * kNanosPerMillisecond;
+    static constexpr int64_t kSecondsPerMicroSecond = 1000000;
```

Comment:
Should be called kMicroSecondsPerSecond.

Reply:
Done!